### DISCUSSION TITLE
Question about Time-Series Model

### DISCUSSION THREAD

**Discussion by frank-dkvan** [ArcadeData/arcadedb#469] [Q&A]
Hi @lvca  would you advice the roadmap  and latest status of Time-Series Model?  I found "coming soon"  in document..don't know when will it get ready? thanks!

**Comment by lvca** [comment]
Hi @frank-dkvan, we have several use cases of using ArcadeDB with time series. One of those is the company's Arcade Trader (algo trading platform) where the team used ArcadeDB to store and manage the time series from stocks. Unfortunately, it's not easy as we thought to create a framework API generic enough to satisfy the different use cases with time series.

We're coming back on this module at least once a month, but we can't still find an elegant approach that would be efficient enough. Custom modeling by using the Multi-Model approach based on the single use-cases is still the best approach so far.

I'm sorry if we're unable to come up with a date or something more precise, but it's still a work-in-progress task.

What could be useful, maybe, it's an open discussion about the best practices and the techniques we found to be efficient with time series?

**Reply by frank-dkvan** [reply]
got it,thanks  @lvca   would you please share the case study or solutions about the time series from stocks... see if we can get inspiration and contribute something  in near future...

**Comment by psytron** [comment]
Still excited about this feature!

**Comment by lvca** [comment]
We're close to starting the official development of the Time Series module after a lot of thoughts and requirements form users. You can followup here: https://github.com/ArcadeData/arcadedb/discussions/1180.
